### PR TITLE
minor style improvements

### PR DESCRIPTION
**Pull request by feruzm**

### CODE REVIEW

File: src/style/_markdown.scss [hunk 1]
```diff
@@ -92,6 +92,9 @@
 
   img {
     max-width: 100%;
+    display: block;
```
Comment (talhasch) [inline src/style/_markdown.scss:95]: @feruzm why are we adding display and margins? i don't remember exactly but this attributes caused issues once.

Comment (feruzm) [reply src/style/_markdown.scss:95]: @talhasch Idea is to center images if they are not and margin auto works best with display block, you could try removing display block and see if it works as expected.

Comment (talhasch) [reply src/style/_markdown.scss]: this basically disallows users to add images aligned to left. removed styles. rest looks good.

Comment (feruzm) [reply src/style/_markdown.scss]: I did check pull-right and pull-left was working as expected and `<a>` style was about video embeds also looks better when they are centered. Centering images are really important if user doesn't use `<center>` or pull-right, pull-left then we should center it with styles.

**Review by talhasch — APPROVED** [review]

**Pull request merged**